packages/webamp-modern-2/src/skin/AudioPlayer.ts

```diff
@@ -131,10 +157,48 @@ export class AudioPlayer {
     return this._audio.currentTime / this._audio.duration;
   }
 
+  getState(): number {
+    if(!!this.__isStop) { // To distinc with pause
```

Comment:
This is already a boolean.
```suggestion
    if(this.__isStop) { // To distinct from pause
```